uiowa/uiowa
# Emergency Hawk Alerts API Tweaks

Relates to https://github.com/uiowa/uiowa/issues/7818




# How to test

```
ddev blt ds --site=emergency.uiowa.edu
```
Login and edit a couple existing alerts. Make some published/complete, unpublished/complete, published/incomplete, unpublished/incomplete. They should already have different created dates.
Go to https://emergency.uiowa.ddev.site/api/active and see that the unpublished content is not included, the complete content is not included and that the ones that are there are sorted with the most recent date first in the list.

The CORS change needs live-testing - See the comments on https://github.com/uiowa/uiowa/issues/7818

The sort is important because Signage only fetches the first one in the API.
